frontend/src/pages/RecordingDetailsPage.tsx

```diff
@@ -1,10 +1,22 @@
-import { EuiFlexGroup, EuiFlexItem, EuiPageTemplate, EuiText } from '@elastic/eui';
-import { useLocation } from 'react-router';
+import {EuiButton, EuiFlexGroup, EuiFlexItem, EuiPageTemplate, EuiText} from '@elastic/eui';
+import {useLocation, useNavigate} from 'react-router';
 
 import prettyBytes from 'pretty-bytes';
+import {useDeleteRecordingById} from "../api";
+import {useQueryClient} from "@tanstack/react-query";
+import {render} from "react-dom";
 
 function RecordingDetailsPage() {
   const location = useLocation()
+  const navigate = useNavigate()
+  const buttonStyle = {
+    marginRight: "25px",
+    marginTop: "25px"
+  }
+
+  const deleteCurrentRecording = () => {
+    navigate('/recordings', {state: {recordingId: location.state.item.id}});
```

Comment:
Same comment as in RecordingDataGrid

Reply:
I am not sure a URL parameter would be appropriate when navigating back to the recording dashboard page.